ui/src/js/common/footer/pages/Financial.js

```diff
@@ -0,0 +1,64 @@
+import React, {
+  Component
+} from 'react';
+import {
+  render
+} from 'react-dom';
+import Boostrap from 'bootstrap';
+import axios from 'axios';
+
+require("./financial.less");
+
+export default class financial extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      financial: []
+    }
+  }
+
+  componentDidMount() {
+    var that = this;
+    axios.get('http://localhost:8082/articles?articleType=FINANCIALS&language=' + localStorage.getItem('language')).then(function(response) {
+      that.setState({
+        financial: response.data
+      });
+    }).catch(function(response) {
+      if (response instanceof Error) {
+        console.error('Error', response.message);
+      } else {
+        console.error(response.data);
+        console.error(response.status);
+        console.error(response.headers);
+        console.error(response.config);
+      }
+    });
+  }
+
+  render() {
+    var that = this;
+    return (
+      <div className="container paddingTopBottom15 financial">
+          <div className="row">
+            <div className="col-md-12">
+              <h1>Finanzen</h1>
+            </div>
+          </div>
+          <div className="row">
+            <div className="col-md-12">
+              <div>
+                {this.state.financial.map(function(about, i) {
```

Comment:
For clarification: Does this about variable has something to do with the about page or is it financial? I recommend to use a name that this describes the content better.